python/grass/jupyter/tests/conftest.py

```diff
@@ -0,0 +1,36 @@
+"""Fixture for ReprojectionRenderer test with simple GRASS location, raster, vector"""
+
+from types import SimpleNamespace
+
+import pytest
+
+import grass.script as gs
+import grass.script.setup as grass_setup
+
+
+@pytest.fixture(scope="module")
+def simple_dataset(tmp_path_factory):
+    """Start a session and create a raster time series
+    Returns object with attributes about the dataset.
+    """
+    tmp_path = tmp_path_factory.mktemp("simple_dataset")
+    location = "test"
+    gs.core._create_location_xy(tmp_path, location)  # pylint: disable=protected-access
+    with grass_setup.init(tmp_path / location):
+        gs.run_command("g.proj", flags="c", epsg=26917)
+        gs.run_command("g.region", s=0, n=80, w=0, e=120, b=0, t=50, res=10, res3=10)
+        # Vector
+        vector_name = "point"
+        full_vector_name = f"{vector_name}@PERMANENT"
```

Comment:
When the value is used only as a parameter in a call where the items are already keyword arguments line by line, it seems to me that removing the variable and doing the work in the keyword argument assignment is more readable because it saves lines and variables one needs to track.